go.mod

```diff
@@ -317,6 +331,7 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/experimentalmetricmetadata v0.111.0 // indirect
 	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl v0.111.0 // indirect
 	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/pdatautil v0.111.0 // indirect
+	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/sampling v0.110.0 // indirect
```

Comment:
why is this dependency still on v0.110.0 ? shouldn't it be on v0.111.0 like all the other otel-collector dependencies ?

Reply:
The components dependency bumps for v0.111.0 were not merged, they should be good now. Thanks for catching this!